Dockerfile

```diff
@@ -26,7 +26,8 @@ RUN \
 
 COPY log4j2.properties /opt/elasticsearch/config/
 COPY elasticsearch.yml /opt/elasticsearch/config/
-COPY jvm.options /opt/elasticsearch/config/
+COPY jvm.options-override /opt/elasticsearch/config/
+RUN cd /opt/elasticsearch/config/ && cat jvm.options-override >> jvm.options && rm jvm.options-override
```

Comment:
This isn't actually working out and I'll revert it. It gets very confusing when you want to customize a property by removing it, or setting a different value for something which is already set.

I prefer being able to see them all in one file.

Reply:
@Sanne As you wish. Did you make sure to use the correct jvm.options file from the new version of Elasticsearch? It changed.

Reply:
Never mind, I can see you did. Sorry for the noise.

Reply:
np, good that you watch :+1: 

BTW the problem was I couldn't un-set the JVM GC tuning; with small heaps & having JDK11 those were not making sense.